common/include/Utilities/PathUtils.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+#include "FixedPointTypes.inl"
+#include "ghc/filesystem.h"
+#include "Path.h"
+
+namespace fs = ghc::filesystem;
+
+class FolderUtils
+{
+protected:
+	fs::path folder;
+
+public:
+	bool CreateFolder(fs::path path);
+	// Is the folder empty
+	bool Empty(std::string);          
+	// Does the path exist
+	bool DoesExist(std::string path);
+	// Does the path exist
+	bool DoesExist(fs::path path);
+};
```

Comment:
What is the purpose of having this as a class?

Reply:
I made that to eventually overwrite https://github.com/PCSX2/pcsx2/blob/0d02d1e41bdb49ecbe6a1740507e3393683d7628/common/include/Utilities/Path.h#L26 Jakes wxDir class.

Reply:
I don't really care what Jake did or didn't do.

I don't see why this can't just be a namespace seeing as the only member you have here is unused (doesn't follow our naming convention anyway) and all the functions here are static in nature. If later you wanna pull it out of a namespace and do something along like lines of [QDir](https://doc.qt.io/qt-5/qdir.html) later I'd be open to that but that's not here and now.

Reply:
Done